lib/rust/ensogl/core/src/system/gpu/shader/compiler.rs

```diff
@@ -171,6 +173,7 @@ pub enum Progress {
 pub struct Jobs {
     compile:              Vec<Job<shader::Code>>,
     link:                 Vec<Job<shader::CompiledCode>>,
+    read_cache:           Vec<Job<()>>,
```

Comment:
`read_cache` sounds like an action, but it is a field. I can't understand what a field with such a name contains.

Reply:
This follows the naming scheme of other job queues. The job queue is named after an action that the job performs. Is my interpretation of those incorrect?